monkestation/code/modules/cybernetics/augments/_base_changes.dm

```diff
@@ -3,7 +3,6 @@
 
 /obj/item/organ/internal/cyberimp
 	var/hacked = FALSE
-	var/syndicate_implant = FALSE //Makes the implant invisible to health analyzers and medical HUDs.
```

Comment:
this var was not used anywhere - nothing set it or checked it - and the "invisible to health analyzers and medical HUDs" thing is already achieved via the `ORGAN_HIDDEN` flag.